configs.d/input/debug.conf

```diff
@@ -0,0 +1,5 @@
+<source>
+  @type debug_agent
```

Comment:
You have to expose port 24230 `docker ... -p 24230:24230`, then use the program `fluent-debug` provided with the fluentd distribution.  It's primarily for plugin developers, or for debugging really hard problems using `druby`.